dspace-api/pom.xml

```diff
@@ -319,8 +319,16 @@
             <artifactId>utilities</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.dspace</groupId>
+            <groupId>net.handle</groupId>
             <artifactId>handle</artifactId>
+            <version>9.3.0</version>
+            <exclusions>
+                <!-- A later version is brought in by google-oauth-client -->
```

Comment:
comments copied from dspace 7, I'm guessing there could be a gson collision, but maybe earlier?